### ISSUE TITLE
Unable to start server on OS X after installing the gem

### ISSUE THREAD

**Issue by ybart** [narkisr-deprecated/vagrant-dns-server#3]
Hi. I'm trying to setup vagrant-dns-server but I can't succeed until now.

I'm running OS X 10.8.

Here is what I've tried so far.

```
gem install vagrant-dns-server
```

This failed because of missing dependencies, so I've installed ZeroMQ using homebrew. However it doesn't work with latest version. So I've downgraded ZeroMQ to 2.2.0.

```
cd /usr/local/
git checkout 6a2e6ef /usr/local/Library/Formula/zeromq.rb # Selects v2.2.0
cd -
brew install zmq
```

After that, I succeeded at installing the gem on my Mac.

However running the server failed:

```
$ vagrant_dns server
I, [2013-11-25T00:23:21.510417 #19440]  INFO -- : Starting RubyDNS server (v0.6.0)...
I, [2013-11-25T00:23:21.510507 #19440]  INFO -- : Listening on tcp:localhost:53
no acceptor (port is in use or requires root privileges)
/Users/ybart/.rvm/gems/ruby-2.0.0-p247/gems/eventmachine-1.0.3/lib/eventmachine.rb:526:in `start_tcp_server'
/Users/ybart/.rvm/gems/ruby-2.0.0-p247/gems/eventmachine-1.0.3/lib/eventmachine.rb:526:in `start_server'
/Users/ybart/.rvm/gems/ruby-2.0.0-p247/gems/rubydns-0.6.0/lib/rubydns.rb:82:in `block (2 levels) in run_server'
/Users/ybart/.rvm/gems/ruby-2.0.0-p247/gems/rubydns-0.6.0/lib/rubydns.rb:77:in `each'
/Users/ybart/.rvm/gems/ruby-2.0.0-p247/gems/rubydns-0.6.0/lib/rubydns.rb:77:in `block in run_server'

...

```

However, running as root does not help, since it will not be able to find the gem:

```
$ sudo vagrant_dns server
Password:
/Users/ybart/.rvm/rubies/ruby-2.0.0-p247/lib/ruby/site_ruby/2.0.0/rubygems/dependency.rb:296:in `to_specs': Could not find 'vagrant-dns-server' (>= 0) among 0 total gem(s) (Gem::LoadError)
    from /Users/ybart/.rvm/rubies/ruby-2.0.0-p247/lib/ruby/site_ruby/2.0.0/rubygems/dependency.rb:307:in `to_spec'
    from /Users/ybart/.rvm/rubies/ruby-2.0.0-p247/lib/ruby/site_ruby/2.0.0/rubygems/core_ext/kernel_gem.rb:47:in `gem'
    from /Users/ybart/.rvm/gems/ruby-2.0.0-p247/bin/vagrant_dns:22:in `<main>'
```

How should I proceed to successfully run the server ?

**Comment by narkisr** [comment]
See latest usage example, this plugin was tested only on Ubuntu so take that into account

Thanks

**Comment by ybart** [comment]
Thanks, I will give it a try ASAP.

**Comment by ybart** [comment]
I was busy these days. I keep you updated.

**Comment by narkisr** [comment]
Cool no problem

**Comment by ybart** [comment]
For now, I have trouble installing the plugin. The embedded Vagrant ruby could not find the zmq lib installed by Homebrew to properly compile zmq gem native extensions. 

The problem is related to embedded Vagrant ruby because I successfully installed the gem with the system Ruby.

For the records: I used OS X Mavericks, and installed ZeroMQ 2.2.0 using Homebrew (with brew versions to select a compatible version, as advertised here: http://stackoverflow.com/questions/14781751/installing-zmq-gem-on-mountain-lion ). Installed Vagrant version is 1.3.5
